src/main/java/org/reaktivity/command/log/internal/LoggableRoutes.java

```diff
@@ -153,16 +153,15 @@ private String extension(
         }
         else if ("http2".equals(nukleusName))
         {
-            HttpRouteExFW exFW = new HttpRouteExFW();
+            HttpRouteExFW httpExt = new HttpRouteExFW();
             final int index = route.extension().offset();
-            exFW.wrap(route.extension().buffer(), index, index + route.extension().sizeof());
-            HttpHeaderFW authority = exFW.headers().matchFirst(h -> h.name().asString().equals(":authority"));
+            httpExt.wrap(route.extension().buffer(), index, index + route.extension().sizeof());
+            HashMap<String, String> headers = new HashMap<>();
+            httpExt.headers().forEach(h -> headers.put(h.name().asString(), h.value().asString()));
             extension = String.format(
                     "{" +
-                    "\"headers\":{\"%s\": \"%s\"}" +
-                    "}",
-                     authority.name().asString(),
-                     authority.value().asString());
+                    "\"headers\": %s" +
+                    "}", headers.toString());
```

Comment:
Build up the string without storing them all in a map first.

Reply:
OK sure but I thought this is much easer

Reply:
What is the benefit of building it into a string?

Reply:
Avoids the intermediate overhead of the map and also produces proper JSON.